integration-tests/gradle/src/test/java/io/quarkus/gradle/QuarkusGradleWrapperTestBase.java

```diff
@@ -50,7 +50,7 @@ public BuildResult runGradleWrapper(File projectDir, String... args) throws IOEx
 
         boolean done = p.waitFor(5, TimeUnit.MINUTES);
         if (!done) {
-            p.destroy();
+            p.destroyForcibly().waitFor();
```

Comment:
Maybe we should use something like https://github.com/quarkusio/quarkus/blob/f7e6dec34114331542f52669fb42ed77c5ff9f34/test-framework/common/src/main/java/io/quarkus/test/common/LauncherUtil.java#L89 ?

Reply:
Yes that will be better, the process could stop gracefully with that. I will make that method public and use it. I wasn't aware of it.

Reply:
Or should I copy/paste it?

Reply:
Probably just copy and paste for now. We can always move it to some common location later